src/NUnitFramework/framework/Internal/PlatformHelper.cs

```diff
@@ -245,6 +246,13 @@ public bool IsPlatformSupported( string platform )
                 case "WIN8.1":
                     isSupported = os.IsWindows81;
                     break;
+                case "WINDOWS10":
+                case "WIN10":
+                    isSupported = os.IsWindows10;
+                    break;
+                case "WINDOWSSERVER":
```

Comment:
I was thinking the same, but wasn't sure. I will change it. Let me know if
the rest looks OK and I will merge when I update.
On Jun 9, 2015 10:12 PM, "CharliePoole" <notifications@github.com> wrote:

> In src/NUnitFramework/framework/Internal/PlatformHelper.cs
> <https://github.com/nunit/nunit/pull/680#discussion_r32088681>:
>
> > @@ -245,6 +246,13 @@ public bool IsPlatformSupported( string platform )
> >                  case "WIN8.1":
> >                      isSupported = os.IsWindows81;
> >                      break;
> > +                case "WINDOWS10":
> > +                case "WIN10":
> > +                    isSupported = os.IsWindows10;
> > +                    break;
> > +                case "WINDOWSSERVER":
>
> I think consistency with our own terminology may be more important than
> conforming to MS terminology here. WindowsServer in the context of
> PlatformAttribute sounds like it should include all prior versions of
> server as well. I'd go with Windows10Server or WindowsServer10.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/680/files#r32088681>.
>